src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_strategy_selection_test.py

```diff
@@ -97,6 +97,16 @@ def setUp(self):
     data.append(strategy3)
     ndb.put_multi(data)
 
+    query = data_types.FuzzStrategyProbability.query()
+    distribution = []
+    for strategy_entry in list(ndb_utils.get_all_from_query(query)):
+      distribution.append({
+          "strategy_name": strategy_entry.strategy_name,
+          "probability": strategy_entry.probability
+      })
+    environment.set_value('USE_BANDIT_STRATEGY_SELECTION', True)
+    environment.set_value('STRATEGY_SELECTION_DISTRIBUTION', distribution)
+
   def test_multi_armed_bandit_strategy_pool(self):
     """Ensures a call to the multi armed bandit strategy selection function
     doesn't yield an exception."""
```

Comment:
should we be updating this test to actually check the results?

Reply:
the tests below this one test the deterministic behavior of this pool generation process. because the pool generated by this call is non-deterministic, i simply tested for an exception for this one.